added out going video box resizing

mohamed-ahmed opened the pull request:
added out going video box resizing similar to Skype video calls

Commit:
added out going video box resizing

added out going video box resizing similar to Skype video calls

ryanseys commented:
Very nice! Can I request a slight clean up of the code to include brackets `{` `}` on the if statements and `;` for the line endings? Just an additional push to your `master` works. Thanks!

Commit:
emails added to cache open entry

need to add email buttons to DOM, check if emails are valid, and test. I
wrote this while tired. will check in morning

Commit:
each invite sent adds a shortcut button to invite the person later

Every time you invite someone, their email will be saved on the page,
when accessing it through same device and browser through its cookies.
Good for testing, and end-user.

cleaned up prev commit

Commit:
newline added
